style/client.css

```diff
@@ -3092,7 +3092,38 @@ a.ilink.yours {
 	height: 16px;
 	font-size: 9pt;
 }
-
+.selector:not(*:root), input[type=range]{
```

Comment:
I can't tell what `.selector` is doing here. We don't seem to have any element with `class="selector"`?

Reply:
I noticed my changes were breaking on Firefox (and potentially other browsers). This snippet makes it only work on Chrome. I'd be lying if I said I understand what it's doing exactly, I found it on stackoverflow or something. It works though.

Reply:
Hm, I'll try to address Firefox separately.

Reply:
Thanks for explaining it to me! I ended up going with a more readable workaround for Firefox: df710473067d3856fa4